camel-openmrs/src/main/java/org/openmrs/eip/component/camel/SendShutdownNotificationEmailProcessor.java

```diff
@@ -0,0 +1,84 @@
+package org.openmrs.eip.component.camel;
+
+import java.io.File;
+
+import javax.activation.FileDataSource;
+
+import org.apache.camel.Endpoint;
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.attachment.AttachmentMessage;
+import org.apache.camel.attachment.DefaultAttachment;
+import org.openmrs.eip.component.SyncProfiles;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component("sendShutdownNotificationEmailProcessor")
+public class SendShutdownNotificationEmailProcessor implements Processor {
+	
+	private static Logger log = LoggerFactory.getLogger(SendShutdownNotificationEmailProcessor.class);
+	
+	@Value("${spring.email.attachment.application-log-file}")
+	private String logsURI;
+	
+	@Value("${smtp.host.name}")
+	private String smtpHostName;
+	
+	@Value("${smtp.host.port}")
+	private String smtpHostPort;
+	
+	@Value("${smtp.auth.user}")
+	private String smtpAuthUser;
+	
+	@Value("${smtp.auth.pass}")
+	private String smtpAuthPass;
+	
+	@Value("${spring.email.send-to}")
+	private String springEmailSendTo;
+	
+	@Value("${spring.profiles.active}")
+	private String activeProfile;
+	
+	@Value("${db-sync.senderId:#{null}}")
+	private String dbSyncSenderId;
+	
+	@Value("${db-sync.receiverId:#{null}}")
+	private String dbSyncReceiverId;
```

Comment:
Move this property out of this processor to the receiver route where you call this processor class so that you don't have to make it nullable here